Privacy.md

```diff
@@ -0,0 +1,88 @@
+
+- **have full network access:** The app is capable of creating network sockets and can use customized network protocols. This capability does not require the software to send data to the internet.
+- **view network connections:** This is the capability of the app to view information about network connections such as which networks exist and are connected.
+- **run at startup:** Allows the app to have itself started as soon as your device has finished booting.
+
+## Types of Data Collected
+
+### Personal Data
+The developer doesn't collect personally identifiable data.
+
+### Usage Data
+The developer doesn't collect usage data.
+
+### Installation Data
+The developer distributes this software via channels like F-Droid and GitHub where download counts are not maintained. So the developer has no knowledge or way to acquire knowledge that the user has installed or is using this software. The developer also distributes this software via google play store, but the developer is not privy to this information either.
+
+
+## Crash reports sending
+
+Forecastie is stable and works fine, but you may still experience a few errors and crashes. The information about these crashes is important to the developer so the issues can be fixed immediately and the software is running efficiently again. The developer isn't able to get reports when the app crashes. If you encounter a bug during the use of this software, please open an [issue](https://github.com/martykan/forecastie/issues/new) on GitHub and it will be fixed in due time. When submitting a crash report, please provide a logcat to help the developer pinpoint the cause of the crash quickly and fix it accordingly.
+
+
+## Cookies
+
+Cookies are files with a small amount of data that are commonly used as anonymous unique identifiers. These are sent to your browser from the websites that you visit and are stored on your device's internal memory.
+
+This Service does not use these “cookies” explicitly. However, the app may use third party code and libraries that use “cookies” to collect information and improve their services. You have the option to either accept or refuse these cookies and know when a cookie is being sent to your device. If you choose to refuse our cookies, you may not be able to use some portions of this Service.
+
+## Service Providers
+
+The developer may employ third-party companies and individuals due to the following reasons:
+
+To facilitate our Service;
+
+To provide the Service on our behalf;
+
+To perform Service-related services; or
+
+To assist us in analyzing how our Service is used.
+
+I want to inform users of this Service that these third parties have access to your Personal Information. The reason is to perform the tasks assigned to them on our behalf. However, they are obligated not to disclose or use the information for any other purpose.
+
+## Transfer Of Data
```

Comment:
Do any of the third-party libraries do this? I suspect not, again making this a bit abstract.